packages/desktop-client/src/components/common/Modal.tsx

```diff
@@ -489,3 +492,12 @@ export function ModalCloseButton({ onPress, style }: ModalCloseButtonProps) {
     </Button>
   );
 }
+
+function ErrorFallback() {
+  const { t } = useTranslation();
+  return (
+    <Text style={{ ...styles.mediumText, color: theme.errorText }}>
+      {t('There was a problem loading the modal')}
```

Comment:
We prefer `Trans` for translations within JSX:

```suggestion
      <Trans>There was a problem loading the modal</Trans>
```